### ISSUE TITLE
Setup minimal site with working Pelican build (Jira: LUCENE-9012)

### ISSUE THREAD

**Issue by mocobeta** [mocobeta/sandbox-lucene-10557#179]
---
### Jira information

Original Jira: https://issues.apache.org/jira/browse/LUCENE-9012
Reporter: Jan Høydahl (`@janhoy`)
Assignee: Jan Høydahl (`@janhoy`)
Created: 2019-10-17T09:13:07.981+0000
Updated: 2019-11-11T08:47:13.139+0000
Resolved: 2019-11-11T08:47:13.066+0000

Issue Links:

Sub-Tasks:

Pull Requests:

**Comment by mocobeta** [comment]
I started this ticket to convert the cms site Pelican. So far I was able to get Lucene Core built with parity to the current live site. While doing so I didn't think it was right to take commit history from others and make it look like the entire site was my commit. So I moved over to LUCENE-9013 (#180) first.

I'll continue the Pelican migration once LUCENE-9013 (#180) is merged.

Author: Adam Walz
Created: 2019-10-27T16:04:04.146+0000
Updated: 2019-10-27T16:04:04.146+0000

**Comment by mocobeta** [comment]
Converting the cms site to pelican at <https://github.com/adamwalz/lucene-site/tree/LUCENE-9012 (#179)>

Author: Adam Walz
Created: 2019-10-28T04:34:06.133+0000
Updated: 2019-10-28T04:34:06.133+0000

**Comment by mocobeta** [comment]
I've now converted everything except for the solr pages to Pelican. [Lucene TLP, Lucene Core, PyLucene, PyLucene JCC, Open Relevance]

Author: Adam Walz
Created: 2019-10-30T07:28:39.928+0000
Updated: 2019-10-30T07:28:39.928+0000

**Comment by mocobeta** [comment]
`@janhoy`  Now that the cms version history is merged into github, this PR is ready to be reviewed <https://github.com/apache/lucene-site/pull/1>

Author: Adam Walz
Created: 2019-10-30T14:47:50.603+0000
Updated: 2019-10-30T14:47:50.603+0000

**Comment by mocobeta** [comment]
Merged PR 1, please see <https://github.com/apache/lucene-site> now. The site (except Solr) can now be built and viewed locally following the steps in README.

Next is solr parts, and then general cleanup (dead links, restructure as needed etc). We can do that as followup PRs in this same JIRA.

Author: Jan Høydahl (`@janhoy`)
Created: 2019-10-30T18:42:48.538+0000
Updated: 2019-10-30T18:42:48.538+0000

**Comment by mocobeta** [comment]
`@janhoy`  I've createdA <https://github.com/apache/lucene-site/pull/2> to migrate the solr pages to Pelican. As discussed, future PRs will be needed to go through each page with a more fine-toothed comb and ensure 100% parity with the current production site, as well as to reduce some code duplication introduced in the migration.

Author: Adam Walz
Created: 2019-11-03T20:25:32.114+0000
Updated: 2019-11-03T20:25:32.114+0000

**Comment by mocobeta** [comment]
`@janhoy`  <https://github.com/apache/lucene-site/pull/3> is ready which reduces code duplication by using Pelican extends and include syntax.

git diff is +3,512 −33,736 :D

Author: Adam Walz
Created: 2019-11-07T23:13:01.595+0000
Updated: 2019-11-07T23:13:01.595+0000

**Comment by mocobeta** [comment]
I merged PR 3. Just had a few comments which can be tackled in followups. Guess we can close this Jira now :)

Author: Jan Høydahl (`@janhoy`)
Created: 2019-11-11T08:45:15.528+0000
Updated: 2019-11-11T08:45:15.528+0000
